sample-input/pin-cell/PC-isotope.py

```diff
@@ -0,0 +1,274 @@
+from openmoc import *
+import openmoc.log as log
+import openmoc.plotter as plotter
+from openmoc.options import Options
+import h5py as h5
+import numpy as np
+
+###############################################################################
+#######################   Main Simulation Parameters   ########################
+###############################################################################
+
+options = Options()
+
+num_threads = options.getNumThreads()
+track_spacing = options.getTrackSpacing()
+num_azim = options.getNumAzimAngles()
+tolerance = options.getTolerance()
+max_iters = options.getMaxIterations()
+
+log.set_log_level('NORMAL')
+
+
+###############################################################################
+###########################   Creating Isotopes    ############################
+###############################################################################
+
+log.py_printf('NORMAL', 'Importing isotope data from HDF5...')
+
+mgxs = h5.File('mgxs-8-groups.h5')
```

Comment:
just add a comment clarifying that you are importing microscopic cross-sections from HDF5 to populate your ``Isotopes``